# NetSquare.msg

# Net square corner positions
geometry_msgs/Point32[4] points